File: movies-api/src/main/java/com/ivanfranchin/moviesapi/userextra/UserExtraRepository.java
package com.ivanfranchin.moviesapi.userextra;

import com.ivanfranchin.moviesapi.userextra.model.UserExtra;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserExtraRepository extends MongoRepository<UserExtra, String> {
}

File: movies-api/src/main/kotlin/com/ivanfranchin/moviesapi/userextra/UserExtraRepository.kt
package com.ivanfranchin.moviesapi.userextra

import com.ivanfranchin.moviesapi.userextra.model.UserExtra
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface UserExtraRepository : MongoRepository<UserExtra, String>
